.scss-lint.yml

```diff
@@ -83,7 +83,7 @@ linters:
     extra_properties: []
   QualifyingElement:
     enabled: true
-    allow_element_with_attribute: false
+    allow_element_with_attribute: true
```

Comment:
Mhh. I didn’t think about this, but I’m also entirely OK styling things based on their attribute…

What do you reckon @petronbot?